go/vt/schemadiff/errors.go

```diff
@@ -282,10 +282,33 @@ type ForeignKeyDependencyUnresolvedError struct {
 }
 
 func (e *ForeignKeyDependencyUnresolvedError) Error() string {
-	return fmt.Sprintf("table %s has unresolved/loop foreign key dependencies",
+	return fmt.Sprintf("table %s has unresolved foreign key dependencies",
 		sqlescape.EscapeID(e.Table))
 }
 
+type ForeignKeyLoopError struct {
+	Table string
+	Loop  []string
```

Comment:
Is it useful to have for each step in the loop also the column name? To more exactly identify the specific loop?

Reply:
I'm afraid of clutter. The foreign key might reference multiple columns. It is indeed possible that same table `child` will reference table `parent` with two different foreign keys, one of which has a cycle, the other does not. In such case it would indeed be beneficial to identify the referenced columns both on parent and child; but again this is an amount of information that is likely to create much background noise. Alternatively, we could use the foreign key name participating in the loop.
Either solution would complicate the loop detection logic only slightly; question is how useful vs. confusing would the information be.

Reply:
Just took a look at the code. The current approach (only report table name) is consistent with other `schemadiff` error reports, that suffice with table name or column name without drilling into specific details. I think this should be fine as it is for now.